apis/r/R/Init.R

```diff
@@ -8,16 +8,6 @@
     ## create a slot for somactx in per-package enviroment, do no fill it yet to allow 'lazy load'
     .pkgenv[["somactx"]] <- NULL
 
-    rpkg_lib <- tiledb::tiledb_version(compact = FALSE)
-    # Check major and minor but not micro: sc-50464
-    rpkg_lib_version <- paste(rpkg_lib[["major"]], rpkg_lib[["minor"]], sep = ".")
-    soma_lib_version <- libtiledbsoma_version(compact = TRUE, major_minor_only = TRUE)
-    if (rpkg_lib_version != soma_lib_version) {
-        msg <- sprintf("TileDB Core version %s used by TileDB-R package, but TileDB-SOMA uses %s",
-                       sQuote(rpkg_lib_version), sQuote(soma_lib_version))
-        stop(msg, call. = FALSE)
-    }
-
```

Comment:
I think we could consider keeping it as an informational message.  Something like

```r
`packageStartupMessage(msg)        
```

Reply:
https://github.com/single-cell-data/TileDB-SOMA/pull/3006/commits/0af41059d340356536c01e7394fd265c7ef25c1b